app/assets/javascripts/organizations.js

```diff
@@ -10,9 +10,15 @@ $(document).ready(function() {
   if (('.org-address').length >= 1 ) {
     displayMap($('.org-address').text());
   }
+
+  //not set to display none so that the page doesn't jump up
+  $('.flash-notice').delay(1000).animate({ opacity: 0 });
```

Comment:
Yeah, I wasn't sure about that. I thought about it but was thinking that some of the others we might want to stay? Would be easy to target them all if we go that route.